UACloudLibraryServer/Models/UANodesetResult.cs

```diff
@@ -48,8 +48,8 @@ public class UANodesetResult
         [JsonProperty(PropertyName = "version")]
         public string Version { get; set; }
 
-        [JsonProperty(PropertyName = "nodesetCreationTime")]
-        public System.DateTime? CreationTime { get; set; }
```

Comment:
Can we keep the CreationTime property in the GraphQL schema to stay backward compatible?

Reply:
As far as i can see, is that you can use the ObsoleteAttribute to specify a field as Deprecated

Reply:
@BoBiene, I checked and since the CreationTime is a free text field in the where clause, it cannot be marked as deprecated. Are you ok with the change as is in that case?

Reply:
@BoBiene ?